### ISSUE TITLE
Nerf the HP of the Overworld NPCs

### ISSUE THREAD

**Issue by Cruuk** [Tslat/Advent-Of-Ascension#481]
**Describe/name the existing feature this suggestion relates to**
The Overworld NPCs (other than Corrupted traveller?)

**Describe the change or feature you think would be good to see**
IIRC one change with the Overworld NPCs is that using name tags on them actually makes them not despawn now, allowing players to potentially take these NPCs to their base and make them powerful dispensers with said name tags.

Which is why I think it would be fair for them to not be so... tanky. I think it's for NPCs that spawn in structures to have 250 hp, but I always saw the Overworld NPCs as more like travelling merchants, if you get what I mean.
Basically I think it would be better if they had like 30-40 hp, to make them more comparable to regular villagers in survivability, since they're a little more like "drifters" rather than someone living in some big structure.

**Comment by Sagisolar** [comment]
This wouldn't change anything lol, just don't be dumb and hit the NPC. With that you would still easily make them dispensers.

**Comment by DiamondWalker** [comment]
I don't agree tbh. The NPCs are already fairly rare. Imagine finding an NPC and it being killed before you get to it, or dying while you're taking it to your house. Also, what if a mob somehow gets into your house? He could kill all your NPCs.

If this is done, they should at least have a way of defending themselves.

**Comment by Cruuk** [comment]
> 
> 
> I don't agree tbh. The NPCs are already fairly rare. Imagine finding an NPC and it being killed before you get to it, or dying while you're taking it to your house. Also, what if a mob somehow gets into your house? He could kill all your NPCs.
> 
> If this is done, they should at least have a way of defending themselves.

Sagi had a comment here but looks like he deleted it lol. That's what the git comment in this picture was
![image](https://user-images.githubusercontent.com/38159897/54397719-ad59b980-4674-11e9-91bb-0c9cd76df4d4.png)

**Comment by Sagisolar** [comment]
> 
> Sagi had a comment here but looks like he deleted it lol. That's what the git comment in this picture was
> ![image](https://user-images.githubusercontent.com/38159897/54397719-ad59b980-4674-11e9-91bb-0c9cd76df4d4.png)

What is the point of showing this, just edit your suggestion to say what Scimi said

**Comment by DiamondWalker** [comment]
I guess it could work if something like them spawning more in villages or not despawning was done.

**Comment by Immortallis1** [comment]
> 
> 
> > Sagi had a comment here but looks like he deleted it lol. That's what the git comment in this picture was
> > ![image](https://user-images.githubusercontent.com/38159897/54397719-ad59b980-4674-11e9-91bb-0c9cd76df4d4.png)
> 
> What is the point of showing this, just edit your suggestion to say what Scimi said

it's easier to just take a screenshot to prove he said it
